app/build.gradle.kts

```
@@ -0,0 +1,220 @@
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+    id("com.google.gms.google-services")
+    id("com.google.devtools.ksp") version "1.7.0-1.0.6"
+    kotlin("kapt")
+    id("com.google.android.libraries.mapsplatform.secrets-gradle-plugin")
+    id("dagger.hilt.android.plugin")
+    id("kotlin-android")
+    id("kotlin-parcelize")
+    id("kover")
+}
+
+val props = rootDir.loadProperties("signing.properties")
+
+android {
+    compileSdk = 32
+
+    defaultConfig {
+        applicationId = "com.kks.nimblesurveyjetpackcompose"
+        minSdk = 21
+        targetSdk = 32
+        versionCode = 1
+        versionName = "0.1.0"
+
+        testInstrumentationRunner = "com.kks.nimblesurveyjetpackcompose.di.CustomTestRunner"
+        vectorDrawables {
+            useSupportLibrary = true
+        }
+    }
+
+    buildTypes {
+        getByName("debug") {
+            signingConfig = signingConfigs.getByName("debug")
+        }
+        getByName("release") {
+            isMinifyEnabled = true
+            isShrinkResources = true
+            isDebuggable = false
+            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+
+    buildFeatures {
+        compose = true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion = "1.2.0"
+    }
+
+    packagingOptions {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1}"
+        }
+    }
+
+    flavorDimensions("version")
```

This is deprecated 👀 
```suggestion
    flavorDimensions.add("version")
```